components/_previews/_preview--fullhtml.njk

```diff
@@ -1 +1,28 @@
+<!DOCTYPE html>
```

Comment:
I think this was an accidental commit?

Reply:
ah, yeah - we don't need that if the issue I had with the GDPR disclaimer and other notification showing up is now fixed. Will remove and push

Reply:
hmm, which bit was the accident? 

I don't think we need preview--nojs which I added now

Reply:
`_preview--fullhtml` is used by other components, so we shouldn't update that one (well not without considering impacts). -- could you use the default preview, still seems to work ok?

Reply:
I _think_ the only change is whitespace? Will check, fix, and update

Reply:
updated the PR with another push of some codes.